src/styles/components/interactive.scss

```diff
@@ -27,6 +27,9 @@
         background-color: transparentize($color: $green, $amount: 0.6);
         cursor: pointer;
     }
+    i {
+        font-size: 13px !important;
```

Comment:
Not preferable (personal opinion) and !important.

Reply:
This seems extremely !important to me now. So I !will get to it !asap.